cloudant.js

```diff
@@ -75,6 +86,15 @@ function Cloudant(options, callback) {
   // Cloudant Database Functions
   // ===========================
 
+  // https://cloud.ibm.com/docs/services/Cloudant/guides?topic=cloudant-database-partitioning#creating-the-database
+  function createPartitioned(db, callback) {
```

Comment:
Can we not just support the `partitioned` parameter in `db.create` like you did for `python-cloudant`?

Reply:
yeah, can do. how about `nano.db.create(name, [params], [callback])`? i.e.
```js
nano.db.create('mydb', { partitioned: true }, cb);
```

Reply:
That looks good to me